playbook/L3/roles/l3-basic-functionality/tasks/l3-basic-functionality.yml

```diff
@@ -235,7 +231,7 @@
   - debug:
       var: result
   - name: TC5 - Verify bridge does not exist
-    command: bash -c "opx-show-vlan --id {{vlanid}}"
```

Comment:
why the vlanid is removed?

Reply:
root@OPX:~# opx-show-vlan --id 100
Traceback (most recent call last):
  File "/usr/bin/opx-show-vlan", line 199, in <module>
    obj.show_vlans(name)
  File "/usr/bin/opx-show-vlan", line 128, in show_vlans
    self.print_vlan(obj[0])
IndexError: list index out of range